repo.go

```diff
@@ -109,6 +109,8 @@ type CloneOptions struct {
 	Quiet bool
 	// The branch to checkout for the working tree when Bare=false.
 	Branch string
+	// The number of revisions to clone when Bare=false.
```

Comment:
```suggestion
	// The number of revisions to clone.
```

I'm pretty sure `--depth` works for repos that are cloned in bare format.